Solutions/PaloAlto-PAN-OS/Package/createUiDefinition.json

```diff
@@ -0,0 +1,334 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/0.1.2-preview/CreateUIDefinition.MultiVm.json#",
+  "handler": "Microsoft.Azure.CreateUIDef",
+  "version": "0.1.2-preview",
+  "parameters": {
+    "config": {
+      "isWizard": false,
+      "basics": {
+        "description": "<img src=\"https://github.com/Azure/Azure-Sentinel/raw/master/Playbooks/PaloAlto-PAN-OS/PaloAltoCustomConnector/PAN-OS_CustomConnector.png\" width=\"75px\" height=\"75px\">\n\n**Important:** _This Azure Sentinel Solution is currently in public preview. This feature is provided without a service level agreement, and it's not recommended for production workloads. Certain features might not be supported or might have constrained capabilities. For more information, see [Supplemental Terms of Use for Microsoft Azure Previews](https://azure.microsoft.com/support/legal/preview-supplemental-terms/)._\n\n**Note:** _There may be [known issues](https://aka.ms/sentinelsolutionsknownissues) pertaining to this Solution, please refer to them before installing._\n\n[PAN‑OS](https://docs.paloaltonetworks.com/pan-os.html) is the software that runs all Palo Alto Networks next-generation firewalls. By leveraging the key technologies that are built into PAN‑OS natively -- App‑ID, Content‑ID, Device-ID, and User‑ID -- you can have complete visibility and control of the applications in use across all users and devices in all locations all the time.\n\nAzure Sentinel Solutions provide a consolidated way to acquire Azure Sentinel content like data connectors, workbooks, analytics, and automations in your workspace with a single deployment step.\n\n**Custom Azure Logic Apps Connectors:** 1, **Playbooks:** 4\n\n[Learn more about Azure Sentinel](https://aka.ms/azuresentinel) | [Learn more about Solutions](https://aka.ms/azuresentinelsolutionsdoc)",
```

Comment:
Hi @NikTripathi, In description replace Azure Sentinel with Microsoft Sentinel and also logo needs .svg file